src/modules/45-dashboards/pages/DashboardsContext.tsx

```diff
@@ -5,17 +5,18 @@
  * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
  */
 
-import React, { useState, useContext } from 'react'
+import React, { useState, useContext, useMemo } from 'react'
 import { useParams } from 'react-router-dom'
 import type { Breadcrumb } from '@harness/uicore'
 import type { AccountPathProps } from '@common/interfaces/RouteInterfaces'
 import { PermissionIdentifier } from '@rbac/interfaces/PermissionIdentifier'
-import { FolderModel, useSearchFolders } from 'services/custom-dashboards'
+import { FolderModel, useSearchFolders, useGetModelTags } from 'services/custom-dashboards'
 
 export interface DashboardsContextProps {
   breadcrumbs: Breadcrumb[]
   includeBreadcrumbs: (breadcrumbs: Breadcrumb[]) => void
   editableFolders: FolderModel[]
+  modelTags?: string[]
```

Comment:
It looks from further down that this will always be passed and will default to an empty array, so you might be able to simplify things by making it required.
```suggestion
  modelTags: string[]
```

Reply:
Indeed but I did this as a trick in an attempt to get coverage above the threshold, since I've given up on that hope I can change it back as you suggested.

Reply:
I find coverage harder to get when introducing branching. Sane defaults _typically_ simplify the code and mean that you have to do less conditions. Doesn't always work, but every time you use `?`, the Istanbul coverage stuff sees that as a branch and will whinge if you don't code for it.

Reply:
Looking at the coverage report, your code is doing just fine... it's the LicenseStore that's killing you. We're well within FM territory I think.

<img width="1024" alt="Screenshot 2023-03-31 at 10 58 58" src="https://user-images.githubusercontent.com/7167253/229089731-22bab786-ff99-4d62-a10f-f2460fc89301.png">

Reply:
I've switch it back to required and the module branch coverage dropped 1%
I think it is due to an annoying side effect where one of the higher level pages (HomePage) is rendering the component that uses this (ModuleTagsFilter). The coverage for ModuleTagsFilter is 100% but coverage for HomePage is lower due to it's complexity.